extensions/undertow-websockets/deployment/src/main/java/io/quarkus/undertow/websockets/deployment/UndertowWebsocketProcessor.java

```diff
@@ -124,10 +126,29 @@ public ServletContextAttributeBuildItem deploy(final CombinedIndexBuildItem inde
                 new ReflectiveClassBuildItem(true, false, annotated.toArray(new String[annotated.size()])));
         reflection.produce(new ReflectiveClassBuildItem(false, false, JsrWebSocketFilter.class.getName()));
 
+        final Collection<AnnotationInstance> serverEndpoints = index.getAnnotations(SERVER_ENDPOINT);
```

Comment:
This should also process client endpoints

Reply:
done, I also added tests for client and server *coders